markdown/preface.markdown

```diff
@@ -11,69 +11,76 @@ Elegant SciPy goes further.
 More than just teaching you how to write code that works, we will inspire you to write code that rocks!
 
 In The Rosie Effect (hilarious book; go read its prequel [The Rosie Project](https://en.wikipedia.org/wiki/The_Rosie_Project) when you’re done with Elegant SciPy), Graeme Simsion twists the conventions of the word "elegant" around.
-Most would use it to describe the visual simplicity of something or someone stylish or graceful.
-Say, the first iPhone.
-Instead, our hero, Don Tillman, uses a computer algorithm to *define* elegance.
-We hope that you will know exactly what he means after reading this book.
+Most would use it to describe the visual simplicity, style, and grace of, say, the first iPhone.
```

Comment:
@stefanv I think this is what Nelle was referring to somehow. The point is that the text of the preface already said this.

As to the point itself, I think it's pretty uncontroversial in industrial design circles that iPhone 1 was a tour de force, and remains the template on which nearly all modern smartphones are built.